docs/setup/setup.md

```diff
@@ -191,6 +194,17 @@ cd $GOPATH/src/github.com/kubeedge/kubeedge
 ```
 #### Run Edge
 
+##### Configuring MQTT mode
+
+The Edge part of KubeEdge uses MQTT for communication between deviceTwin and devices. KubeEdge supports 3 MQTT modes:
+1) internalMqttMode: internal mqtt broker is enabled.
+2) bothMqttMode: internal as well as external broker are enabled.
+3) externalMqttMode: only external broker is enabled.
+
+Use mode field in [edge.yaml](https://github.com/kubeedge/kubeedge/blob/master/edge/conf/edge.yaml#L4) to select the desired mode.
+
+To use KubeEdge in double mqtt or external mode, you need to make sure that [mosquitto](https://mosquitto.org/) or [emqx edge](https://www.emqx.io/downloads/edge) is installed on the edge node as an MQTT Broker.
+
```

Comment:
Could we use these lines replace the L307-308?

Reply:
> If you have run mosquitto on your edge host, please set mqtt.mode to 2

Is that meaning mqtt.mode must be set to 2 in this case?  If not, we can remove this statement. 

> To use KubeEdge in double mqtt or external mode (set mqtt.mode to 1), you need to make sure that mosquitto or emqx edge is installed on the edge node as an MQTT Broker.

This is duplicated, we can just remove that one.

Reply:
Yes, just delete them.